test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common.lua

```diff
@@ -0,0 +1,118 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
```

Comment:
Do we really need such restrictions?

Reply:
@dboltovskyi It is necessary, because the proposal indicates that new parameters are available from a certain version (since = "X.x). Thus, we show that new parameters are available to us with version 6.2.

Reply:
@HSavynetska In this way why we have `minorVersion = 0`?

Reply:
@dboltovskyi  
**minorVersion** was updated from 0 to 2 on the commit 72cc305726e4ddcc2908ed268020954e24e112ce